app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt

```diff
@@ -123,6 +120,8 @@ interface ViewComponent {
 
     fun inject(fragment: GetInTouchFragment)
 
+    fun inject(fragment: AboutSafeFragment)
```

Comment:
Where is this requirement coming from? Is there a ticket for this?

Reply:
same settings structure as in ios app

Reply:
![image](https://user-images.githubusercontent.com/17750984/151191962-fde29825-338b-4223-8994-fda958e25a9e.png)